pkglistgen/tool.py

```diff
@@ -131,6 +131,34 @@ def write_group_stubs(self):
                     x = ET.tostring(x, pretty_print=True, encoding='unicode')
                     fh.write(x)
 
+    def write_productcompose(self):
+        self._check_supplements()
+        summary = dict()
+        archs = ['*'] + self.all_architectures
+        # a single file covering all builds via multibuild flavors
+        with open(os.path.join(self.productcompose_dir, 'default.productcompose'), 'a') as opfh:
+            opfh.write("packagesets:\n")
```

Comment:
Is this YAML? If so, using a proper yaml lib might be better

Reply:
maybe, on the other hand it might be also bloat for such simple operations.